packages/web/src/pages/sign-up-page/pages/CreateEmailPage/CreateEmailPage.tsx

```diff
@@ -21,11 +29,13 @@ export type SignUpEmailValues = {
   email: string
 }
 
+export const emailSchema = z
+  .string({ required_error: messages.invalidEmail })
+  .regex(EMAIL_REGEX, { message: messages.invalidEmail })
+
 const FormSchema = toFormikValidationSchema(
   z.object({
-    email: z
-      .string({ required_error: messages.invalidEmail })
-      .regex(EMAIL_REGEX, { message: messages.invalidEmail })
+    email: emailSchema
```

Comment:
should it be required? what about password?

Reply:
it is required - iirc all zod fields are required by default :D 
the password requirement is not described by a zod schema because the password requirements/validation UI is very particular. so the password (or "login details") page has a custom form validator.

Reply:
Ah, awesome, thank you! Makes sense